astroplan/utils.py

```diff
@@ -92,9 +92,7 @@ def _get_IERS_A_table(warn_update=14*u.day):
     if IERS_A_in_cache():
         table = iers.IERS_Auto.open()
         # Use polar motion flag to identify last observation before predictions
-        index_of_last_observation = ''.join(table['PolPMFlag_A']).index('IP')
-        time_of_last_observation = Time(table['MJD'][index_of_last_observation],
-                                        format='mjd')
+        time_of_last_observation = Time(table['MJD'].max(), format='mjd')
```

Comment:
for an IERS_A table, the original logic gives you the time of the last _observation_ while the new logic gives you the end of the table which is a _prediction_. you want to check for the `PolPMFlag_A` column and, if it exists, use the original logic while falling back to `table['MJD'].max()` if it doesn't.